sysvabi64/sysvabi64.rst

```diff
@@ -984,9 +987,15 @@ static linker and the dynamic linker. There are a number of entries
 that help locate various dynamic relocation tables and other aspects
 that are architecture independent.
 
+The generic dynamic tags are defined in SCO-ELF_ Dynamic Section.
+
 The AArch64 specific dynamic tags are defined in AAELF64_ Dynamic
 Section.
 
+The generic tag ``DT_PLTGOT`` tag has a processor specific
+implementation. On AArch64 it is defined to be the address of the
+``.plt.got`` section.
```

Comment:
I think a typo here -- "generic tag `DT_PLTGOT` tag"  -- has duplication of the word `tag` .
Also typo `.plt.got -> .got.plt`.

Reply:
Thank you! My apologies, if only my eyes saw what I wrote and not what I think I wrote.
Will fix.